JavaKotlinPracticals/app/src/main/java/com/example/javakotlinpracticals/ExceptionHandling.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.javakotlinpracticals
+
+import java.lang.ArithmeticException
+import java.lang.Exception
+
+fun main()
+{
+    //simple try catch
+    try
+    {
+        var a= arrayOf(10,20,30,0)
+        //var c=a[0]/a[3]
+        a.set(4,50)
+    }
+    catch (e:Exception)
+    {
+        println(e)
+    }
+    /*
+    catch (e:ArithmeticException)
+    {
+        println(e)
```

Comment:
If commented code is not needed then always remove it in PR ( if commented code has any special purpose then mention the reason with comment )